src/com/nutrons/framework/producers/Serial.java

```diff
@@ -0,0 +1,57 @@
+package com.nutrons.framework.producers;
+
+import static com.nutrons.framework.util.FlowOperators.toFlow;
+
+import edu.wpi.first.wpilibj.SerialPort;
+
+import io.reactivex.Flowable;
+
+/**
+ * A wrapper around WPI's SerialE class which provides Flowables for data sent to the roboRIO over serial
+ */
+public class Serial {
+    private final SerialPort serial;
+    private final int bufferSize;
+    private final int packetLength;
+    private final Flowable<byte[]> dataStream;
+    private final char terminationCharacter;
+
+    public Serial(int bufferSize, int packetLength, char terminationCharacter) {
+        this(9600, SerialPort.Port.kUSB, bufferSize, packetLength, '\n');
+    }
+
+    public Serial(SerialPort.Port port, int bufferSize, int packetLength, char terminationCharacter) {
+        this(9600, port, bufferSize, packetLength, terminationCharacter);
```

Comment:
Does the baud rate need to be constant, or is it just here to match the arduino?